frontend/app/models/folder.js

```diff
@@ -3,6 +3,11 @@ import Model, { attr, hasMany, belongsTo } from "@ember-data/model";
 export default class Folder extends Model {
   @attr("string") name;
   @attr("string") description;
+  @attr("number") unreadTransferredFiles;
```

Comment:
maybe use a generalized variable name, because this is also used by credentials later, not only files, maybe use "unreadTransfersCount" or something similar to that